zerver/outgoing_webhooks/__init__.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+from typing import Any, Dict
+from six import text_type
+
+from zerver.outgoing_webhooks.generic import GenericBot
+from zerver.outgoing_webhooks.isitup import IsItUpBot
+
+AVAILABLE_OUTGOING_WEBHOOKS = {
+    "GenericBot": GenericBot,
+    IsItUpBot.full_name: IsItUpBot,
```

Comment:
@timabbott That piece of code was mostly for debugging purposes. Pretty sure its safe to just write `"IsItupBot": IsItUpBot`